packages/core/core/src/requests/AssetGraphRequest.js

```diff
@@ -37,6 +37,7 @@ import ThrowableDiagnostic from '@parcel/diagnostic';
 import AssetGraph from '../AssetGraph';
 import {PARCEL_VERSION} from '../constants';
 import createEntryRequest from './EntryRequest';
+import {envCache} from '../Environment';
```

Comment:
feels a bit leaky to export the entire cache map. maybe something like `getEnvironmentById`?